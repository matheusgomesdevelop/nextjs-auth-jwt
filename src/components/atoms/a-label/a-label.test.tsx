import {render, screen} from "@testing-library/react";
import "@testing-library/jest-dom";


// Atoms
import { ALabel } from "..";

describe("Deve renderizar o a-label corretamente", () => {
    it("Deve renderizar o a-label com texto", () => {
        // Arrange
        const text = "Texto do a-label";
        render(<ALabel>{text}</ALabel>);

        // Act
        const get_label = screen.getByTestId("a-label");

        // Assert
        expect(get_label).toBeInTheDocument();
        expect(get_label.textContent).not.toHaveLength(0);
    })
});