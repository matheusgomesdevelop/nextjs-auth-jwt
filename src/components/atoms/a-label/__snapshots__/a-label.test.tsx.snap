// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deve renderizar o a-label corretamente 1`] = `
<label
  className="a_label"
  data-testid="a-label"
/>
`;
